### DISCUSSION TITLE
'display_removed' signal - Getting the space ids related to the display removed

### DISCUSSION THREAD

**Discussion by pabroux** [asmvik/yabai#1988] [Ideas]
Hello,

Yabai is a great and powerful tool. I have been using it since 2020. So I would like first to say thank you.

Recently, I have been playing with `yabai -m signal`. It would be great if the `display_removed` signal could give us (passes) the space ids from the display removed in order to treat them (e.g. removing the padding associated). Currently, it only gives the display ID I'm focusing.

Globally, I think it would be great to get more data from signals.
What do you think about it?

Best,
Sulfyderz.

**Comment by asmvik** [comment]
Hm I'd have to think about how to make this information available, but it doesn't sound impossible.
